<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Repository</class>
 <widget class="QMainWindow" name="Repository">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAddTop_">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="iconSize">
             <size>
              <width>512</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveTop_">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnCopyTop_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Copy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnMoveTop_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Move</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearchTop_">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCompareTop_">
            <property name="text">
             <string>Compare by bytes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="topTabWidget_">
        <property name="styleSheet">
         <string notr="true">QTabWidget
{
border: none;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="topLDirTab_">
         <attribute name="title">
          <string>LDir</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTreeView" name="topLDirTree"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="topDbTab_">
         <attribute name="title">
          <string>DB</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QTableView" name="topDbTable"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAddBot_">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveBot_">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnCopyBot_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Copy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnMoveBot_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Move</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearchBot_">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCompareBot">
            <property name="text">
             <string>Compare by bytes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="bottomTabWidget_">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="bottomLDirTab_">
         <attribute name="title">
          <string>LDir</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTreeView" name="bottomLDirTree"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="bottomDbTab_">
         <attribute name="title">
          <string>DB</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableView" name="bottomDbTable"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
